### PR TITLE
Update search.json

### PR DESCRIPTION
**Pull request by fochoa8**
Proper translation of search.json, all checked with Spanish translation. "Tags" translates to "Etiquetas", the short form to say "Labels" in Spanish as well, in both instances, the application of the word is fine given that Ghost is a blog.

Got some code for us? Awesome 🎊!

Please include a description of your change & check your PR against this list, thanks!

- [x] There's a clear use-case for this code change, explained below
- [x] Commit message has a short title & references relevant issues
- [ ] The build will pass (run `yarn test:all` and `yarn lint`)

We appreciate your contribution!

### CODE REVIEW

**Review by cathysarisky — CHANGES_REQUESTED** [review]
Not recommended for merge until all spelling errors and unexpected punctuation are corrected.

**Review by fochoa8 — COMMENTED** [review]
Done.

**Comment by cathysarisky** [comment]
> Done.

No, there remains another spelling error/typo.

**Comment by fochoa8** [comment]
> > Done.
> 
> No, there remains another spelling error/typo.

image

image

image

image

image

image

Thanks,
Fernando Ochoa Olivares.

**Review by cathysarisky — APPROVED** [review]

**Review by fochoa8 — COMMENTED** [review]
Past tense fixed in "No matches found" proper translation is "No se ecnontraron resultados", as well proper translation of "Matches" equals "Resultados" in Spanish.

image

File: ghost/i18n/locales/es/search.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
     "Authors": "Autores",
     "Cancel": "Cancelar",
-    "No matches found": "No se encontraron coincidencias",
+    "No matches found": "No se encontraron resultados",
     "Posts": "Publicaciones",
-    "Search posts, tags and authors": "Buscar publicaciones, etiquetas y autores",
-    "Show more results": "Mostrar más resultados",
+    "Search posts, tags and authors": "Buscar publicaciones, etiquetas y Autores",
+    "Show more results": "Muestra más resultados",
```
Comment (cathysarisky) [inline ghost/i18n/locales/es/search.json:7]: Could you link a major Spanish-language website that uses Muestra for a show more results link?  I'm seeing Mostrar.

File: ghost/i18n/locales/es/search.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
     "Authors": "Autores",
     "Cancel": "Cancelar",
-    "No matches found": "No se encontraron coincidencias",
+    "No matches found": "No se encontraron resultados",
     "Posts": "Publicaciones",
-    "Search posts, tags and authors": "Buscar publicaciones, etiquetas y autores",
-    "Show more results": "Mostrar más resultados",
+    "Search posts, tags and authors": "Buscar publicaciones, etiquetas y Autores",
```
Comment (cathysarisky) [inline ghost/i18n/locales/es/search.json:6]: What grammar rule has Autores being capitalized?  Thanks!

**Review by cathysarisky — CHANGES_REQUESTED** [review]

**Comment by 9larsons** [comment]
Closing as duplicate of https://github.com/TryGhost/Ghost/pull/21246.

**Pull request closed without merging**